sentences/el/light_HassTurnOn.yaml

```diff
@@ -9,3 +9,16 @@ intents:
           domain: light
           name: all
         response: "lights_area"
+
+      # Turn on all lights in the same area as a satellite device
+      - sentences:
+          - "<turn_on> ( άνοιξε | άναψε ) [ τα ] φώτα<in_here>"
+          - "<turn_on>[ τα ] φώτα( on;<in_here>)"
```

Comment:
I doubt that the  'on;<in_here>' part is correct.
I would say that specifically the '**on;**' part needs to be refactored/translated as well - I assume you use the EN language as your template.